tests/riscv64/smoke.sh

```diff
@@ -73,3 +73,10 @@ generate_llvm_cpu_vmfb tosa-rvv \
 
```

Comment:
We should probably drop the lit tests and move this file to `build_tools/cmake/test_riscv64.sh`. They're not really smoke tests, especially now.

Reply:
I will move and rename this test script. However, I do want to keep the lit tests because @dcaballe still wants to perform the numerical correctness check in this script. I have another implementation for https://github.com/iree-org/iree/pull/10314 that is based on `iree_check_test` that I will push out after I land this one.